UPGRADING.rst

```diff
@@ -4,6 +4,12 @@ Upgrading to Graylog 4.0.x
 
 .. _upgrade-from-33-to-40:
 
+Mark 'List of streams' in Streams API als deprecated
```

Comment:
Maybe we should phrase this as `Deprecation of API endpoint for unpaginated listing of streams`?